content/cloud/data-redis.md

```diff
@@ -99,16 +91,24 @@ Once you have logged in to Redislabs click on the `Data Access Control` and `Dat
 
 ## Configuration
 
-Open the Spin application's `spin.toml` file and add an environment configuration value, within the `[[component]]` section. For example:
+Open the Spin application's `spin.toml` file and add an environment configuration value, within the `[component.redis-rust-application]` section. For example:
 
 <!-- @nocpy -->
 
-```bash
+```toml
 environment = { REDIS_ADDRESS = "redis://username:password@redis.cloud.redislabs.com:16675" }
 ```
 
 > Note: As shown above; the format for the `address` is the redis protocol (`redis://`, username, colon (`:`), password, then the at symbol (`@`), database name, colon (`:`) and finally the port number. You can obtain these values from the Redislabs site from the previous step.
 
+You will also need explicitly add the addresses (your redis database endpoint) to the manifest so that the Wasm component is allowed to send network requests to it. The following example that defines `allowed_outbound_hosts` needs to go within the `[component.redis-rust-application]` section, also.
```

Comment:
```suggestion
You will also need explicitly add the addresses (your Redis database endpoint) to the manifest so that the Wasm component is allowed to send network requests to it. The following example that defines `allowed_outbound_hosts` needs to go within the `[component.redis-rust-application]` section, also.
```